libs/application/templates/institution-collaboration/README.md

```diff
@@ -36,4 +36,5 @@ Fetch development secrets
 
 ## Code owners and maintainers
 
+- [Fuglar](https://github.com/orgs/island-is/teams/fuglar)
```

Comment:
You probably want to change the ownership of the collaboration application to your team in this PR.
Its defined here https://github.com/island-is/island.is/blob/main/.github/CODEOWNERS#L126